enrolments/serializers.py

```diff
@@ -8,22 +9,29 @@
 from .models import Session, Class, Enrolment
 from .validators import (
     validate_class_in_session,
+    validate_student_ids_in_family,
 )
 
 
-class SessionListSerializer(serializers.HyperlinkedModelSerializer):
+class ClassListSerializer(serializers.HyperlinkedModelSerializer):
     class Meta:
-        model = Session
+        model = Class
         fields = [
             "id",
             "name",
         ]
 
 
-class ClassListSerializer(serializers.HyperlinkedModelSerializer):
+class SessionListSerializer(serializers.HyperlinkedModelSerializer):
+    classes = ClassListSerializer(many=True)
```

Comment:
it'd be nice if we could refactor this later so we don't fetch this list of classes unnecessarily on the frontend - made [this ticket](https://www.notion.so/uwblueprintexecs/Developer-Hub-30e9b6c82af24054b31acf7c5e822c89?p=63c00aa81cb34250ab78ef19eb4eaee2) to address later

Reply:
that makes sense, we only want to know the classes when we ask for session detail

Reply:
we need them to display the select options in the current enrolment form, so I'll probably make it a separate serializer later